examples/flax/text-classification/run_flax_glue.py

```diff
@@ -552,8 +552,7 @@ def write_eval_metric(summary_writer, eval_metrics, step):
     dropout_rngs = jax.random.split(rng, jax.local_device_count())
 
     train_batch_size = int(training_args.per_device_train_batch_size) * jax.local_device_count()
-    per_device_eval_batch_size = int(training_args.per_device_eval_batch_size)
-    eval_batch_size = per_device_eval_batch_size * jax.device_count()
+    eval_batch_size = int(training_args.per_device_eval_batch_size) * jax.local_device_count()
```

Comment:
this code seem using jax.device_count() but all left code are using jax.local_device_count(), so I change it to jax.local_device_count() as well.